charts/kyuubi/values.yaml

```diff
@@ -297,6 +297,8 @@ serviceMonitor:
   # The endpoints section in a ServiceMonitor specifies the metrics information for each target endpoint.
   # This allows you to collect metrics from multiple Services across your Kubernetes cluster in a standardized and automated way.
   endpoints: []
+  # Additional labels that can be used so ServiceMonitor will be discovered by Prometheus
+  additionalLabels: {}
```

Comment:
simply use "labels"
```suggestion
  labels: {}
```

Reply:
Would be confusing with chart's generated ones right?

Reply:
I think it's fine because it's already under `serviceMonitor`. Like we did for `server.thriftBinary.annotations`

Reply:
thanks, updated